ext/gmp/config.m4

```diff
@@ -3,8 +3,11 @@ PHP_ARG_WITH(gmp, for GNU MP support,
 
 if test "$PHP_GMP" != "no"; then
 
+  MACHINE_INCLUDES=$($CC -dumpmachine)
```

Comment:
Is this cross platform compatible? From a quick look autoconf has stuff like AC_CANONICAL_BUILD/HOST/TARGET, could those be more appropriate?